### ISSUE TITLE
Any plans on updating this to support current versions of langchain?

### ISSUE THREAD

**Issue by CameronVetter** [yasyf/compress-gpt#3]
This is locked to a version of langchain that is about 50 versions old and the internal structure of langchain has changed significantly.  Do you have plans of integrating this to current versions?

**Comment by yasyf** [comment]
@CameronVetter yes, at some point :) feel free to open a PR if you end up getting around to this before I do!
